README.md

```diff
@@ -1,12 +1,14 @@
-# SpatialOS GDK for Unreal Starter Project
+# SpatialOS GDK for Unreal Test Gyms Project 
 
 * **LICENSE:** Use of the contents of this repository is subject to the [license](LICENSE.md)
-* **REPOSITORY:** github.com/spatialos/UnrealGDKStarterProject
+* **REPOSITORY:** github.com/spatialos/UnrealGDKTestGyms
 
-This Starter Project is both a reference project and a starting point for building games on SpatialOS and Unreal Engine 4.  The project uses the `ThirdPersonExampleMap` and a controllable character `AStarterGameCharacter` on a SpatialOS deployment.
+This project is for testing the Unreal GDK. It contains a set of gyms as well as being a minimal project in which to run unit tests.
```

Comment:
Slight re-order to make it super easy to skim through.

```suggestion
This project is for testing functionality using the SpatialOS GDK for Unreal, it:
* is a minimal project which you can run unit tests in, and
* contains a set of gyms.
```
What does a set of gyms consist of? 
(That is what are the directories and files which make up a gym?)